core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -857,7 +857,8 @@ private[spark] object ApiHelper {
   }
 
   def lastStageNameAndDescription(store: AppStatusStore, job: JobData): (String, String) = {
-    val stage = store.asOption(store.stageAttempt(job.stageIds.max, 0)._1)
+    val stageID = if (job.stageIds.isEmpty) 0 else job.stageIds.max
+    val stage = store.asOption(store.stageAttempt(stageID, 0)._1)
```

Comment:
In this case is there actually an attempt 0?

Reply:
Yes! The `jobRow` invokes the method to generate job page. Maybe I can use a `if` to  handle the partition 0 job.